### PR TITLE
Move our code from glide to dep

### LINKED ISSUE #504: Switch from glide to dep
Glide will sooner or later be deprecated in favour of [dep](https://github.com/golang/dep), which will finally be the official go dependency management tool.

Here is more about that: https://github.com/Masterminds/glide/releases/tag/v0.13.0

### PR DESCRIPTION
**Pull request by cynepco3hahue**
Resolves #504 

Signed-off-by: Lukianov Artyom <alukiano@redhat.com>

### CODE REVIEW

**Comment by rmohr** [comment]
@cynepco3hahue nice.

**Comment by rmohr** [comment]
ok to test

**Comment by cynepco3hahue** [comment]
retest this please

**Comment by rmohr** [comment]
retest this please

**Comment by cynepco3hahue** [comment]
Still the same issue with 
```
+ cluster/kubectl.sh --core get pods --no-headers
+ grep -v Running
virt-nodenetwork-289mc                         0/2       Init:CrashLoopBackOff   27        1h
virt-nodenetwork-m9wj3                         0/2       Init:CrashLoopBackOff   27        1h
```

**Comment by rmohr** [comment]
@cynepco3hahue #516 should finally resolve it. I will manually remove these.

**Comment by rmohr** [comment]
retest this please

File: Makefile [hunk 1]
```diff
@@ -34,23 +34,20 @@ clean:
...
+		${HASH} Gopkg.toml > .Gopkg.toml.hash; \
+		${HASH} Gopkg.lock > .Gopkg.lock.hash; \
+	elif [ ! -e .Gopkg.lock.hash ] || [ "`${HASH} Gopkg.lock`" != "`cat .Gopkg.lock.hash`" ]; then \
 		make sync; \
 	fi
 
 sync:
-	glide install --strip-vendor
-	${HASH} glide.lock > .glide.lock.hash
+	dep ensure -no-vendor
```
Comment (davidvossel) [inline Makefile]: Why are you using the "-no-vendor" flag here? I'd expect 'make sync'  to populate the vendor directory.

Comment (cynepco3hahue) [reply Makefile]: :+1: You are right, I needed to use dep `ensure -vendor-only`

**Pull request merged**